mslib/msui/qt5/ui_sideview_options.py

```diff
@@ -269,3 +316,47 @@ def retranslateUi(self, SideViewOptionsDialog):
+        self.label.setText(_translate("SideViewOptionsDialog", " Plot Title Size               "))
+        self.cbtitlesize.setItemText(0, _translate("SideViewOptionsDialog", "default"))
+        self.cbtitlesize.setItemText(1, _translate("SideViewOptionsDialog", "4"))
+        self.cbtitlesize.setItemText(2, _translate("SideViewOptionsDialog", "6"))
+        self.cbtitlesize.setItemText(3, _translate("SideViewOptionsDialog", "8"))
+        self.cbtitlesize.setItemText(4, _translate("SideViewOptionsDialog", "10"))
+        self.cbtitlesize.setItemText(5, _translate("SideViewOptionsDialog", "12"))
+        self.cbtitlesize.setItemText(6, _translate("SideViewOptionsDialog", "14"))
+        self.cbtitlesize.setItemText(7, _translate("SideViewOptionsDialog", "16"))
+        self.cbtitlesize.setItemText(8, _translate("SideViewOptionsDialog", "18"))
+        self.cbtitlesize.setItemText(9, _translate("SideViewOptionsDialog", "20"))
+        self.cbtitlesize.setItemText(10, _translate("SideViewOptionsDialog", "22"))
+        self.cbtitlesize.setItemText(11, _translate("SideViewOptionsDialog", "24"))
+        self.cbtitlesize.setItemText(12, _translate("SideViewOptionsDialog", "26"))
+        self.cbtitlesize.setItemText(13, _translate("SideViewOptionsDialog", "28"))
+        self.cbtitlesize.setItemText(14, _translate("SideViewOptionsDialog", "30"))
+        self.cbtitlesize.setItemText(15, _translate("SideViewOptionsDialog", "32"))
+        self.cbaxessize.setItemText(0, _translate("SideViewOptionsDialog", "default"))
+        self.cbaxessize.setItemText(1, _translate("SideViewOptionsDialog", "4"))
+        self.cbaxessize.setItemText(2, _translate("SideViewOptionsDialog", "6"))
+        self.cbaxessize.setItemText(3, _translate("SideViewOptionsDialog", "8"))
+        self.cbaxessize.setItemText(4, _translate("SideViewOptionsDialog", "10"))
+        self.cbaxessize.setItemText(5, _translate("SideViewOptionsDialog", "12"))
+        self.cbaxessize.setItemText(6, _translate("SideViewOptionsDialog", "14"))
+        self.cbaxessize.setItemText(7, _translate("SideViewOptionsDialog", "16"))
+        self.cbaxessize.setItemText(8, _translate("SideViewOptionsDialog", "18"))
+        self.cbaxessize.setItemText(9, _translate("SideViewOptionsDialog", "20"))
+        self.cbaxessize.setItemText(10, _translate("SideViewOptionsDialog", "22"))
+        self.cbaxessize.setItemText(11, _translate("SideViewOptionsDialog", "24"))
+        self.cbaxessize.setItemText(12, _translate("SideViewOptionsDialog", "26"))
+        self.cbaxessize.setItemText(13, _translate("SideViewOptionsDialog", "28"))
+        self.cbaxessize.setItemText(14, _translate("SideViewOptionsDialog", "30"))
+        self.cbaxessize.setItemText(15, _translate("SideViewOptionsDialog", "32"))
+        self.label_7.setText(_translate("SideViewOptionsDialog", "         Axes Label Size  "))
+
+
+if __name__ == "__main__":
```

Comment:
on a converted ui file we don't have this

Reply:
> on a converted ui file we don't have this usually. This may get added because of an option used?
> 
> ```
>  pyuic5 ui_sideview_options.ui > ../qt5/ui_sideview_options.py
> ```

may be...i don't know exactly how that came.

Reply:
these `__main__` lines are never get into a test coverage but likly are counted as lines of the project.

Reply:
> these `__main__` lines are never get into a test coverage but likly are counted as lines of the project.

Can/should we delete the `__main__`  lines manually?

Reply:
eventually the command has different options on windows. maybe there is a supress option?

Reply:
Have to study about it.